### PR TITLE
Add footer

### PR DESCRIPTION
**Pull request by maukoquiroga**
Partial solution for #29 

capture d ecran 2017-10-02 a 14 54 56

### CODE REVIEW

File: lib/transport_web/templates/layout/_footer.html.eex [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<footer role="contentinfo">
+  <div class="divider">~</div>
```
Comment (l-vincent-l) [inline lib/transport_web/templates/layout/_footer.html.eex]: You might want to use &nbsp;

File: lib/transport_web/templates/layout/_footer.html.eex [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+  <%= img_tag("/images/logo-footer.svg", alt: "transport.beta.gouv.fr") |> link(to: "/", class: "logo") %>
+
+  <ul class="footer-links">
+    <li>
+      <a class="footer-links-contact" href="mailto:contact@transport.beta.gouv.fr" role="link">
```
Comment (l-vincent-l) [inline lib/transport_web/templates/layout/_footer.html.eex]: the contact address should be a variable

File: lib/transport_web/templates/layout/_footer.html.eex (hunk 1 above)
Comment (maukoquiroga) [reply lib/transport_web/templates/layout/_footer.html.eex]: ?

**Comment by maukoquiroga** [comment]
@l-vincent-l RTM?

File: lib/transport_web/templates/layout/_footer.html.eex (hunk 1 above)
Comment (l-vincent-l) [reply lib/transport_web/templates/layout/_footer.html.eex]: héhé.
You might want to use `&nbsp;`

Comment (maukoquiroga) [reply lib/transport_web/templates/layout/_footer.html.eex]: No, I actually want that symbol, or a text if you have a proposition.

**Comment by maukoquiroga** [comment]
Rebase

File: lib/transport_web/templates/layout/_footer.html.eex (hunk 1 above)
Comment (maukoquiroga) [reply lib/transport_web/templates/layout/_footer.html.eex]: Ah, got it. Done.

File: client/stylesheets/components/_buttons.scss [hunk 3]
```diff
@@ -14,13 +13,13 @@
 [role='link'] {
   &.button-success {
     @extend %button;
-    $green: rgb(28, 184, 65) !default;
-    background: $green;
+    $button-color: $green !default;
+    background: $button-color;
```
Comment (l-vincent-l) [inline client/stylesheets/components/_buttons.scss]: either do 
```
$button-color-success: $green !default;
background: $button-color-success;
```

or
```
background: $green !default;
```

File: client/stylesheets/components/_buttons.scss [hunk 4]
```diff
@@ -14,13 +13,13 @@
...
   }
 
   &.button-warning {
     @extend %button;
-    $red: rgb(223, 117, 20) !default;
-    background: $red;
+    $button-color: $red !default;
+    background: $button-color;
```
Comment (l-vincent-l) [inline client/stylesheets/components/_buttons.scss]: either do 
```
$button-color-warning: $red !default;
background: $button-color-warning;
```

or
```
background: $red !default;
```

File: client/stylesheets/components/_buttons.scss (hunk 3 above)
Comment (maukoquiroga) [reply client/stylesheets/components/_buttons.scss]: Done
done

File: client/stylesheets/components/_buttons.scss (hunk 4 above)
Comment (maukoquiroga) [reply client/stylesheets/components/_buttons.scss]: done

**Pull request merged**